### DISCUSSION TITLE
[27 giugno 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#251] [Esami]
Progettare il circuito sequenziale oppure l'automa (vedi note in fondo) che riceve in ingresso due input a, b che codificano i caratteri J, K, L come segue:

<div align="center">

<table>
  <tr align="center">
    <th>$\Large a,b$</th>
    <th>carattere</th>
  </tr>
  <tr align="center">
    <td>$\Large 00$</td>
    <td>$\Large \texttt{J}$</td>
  </tr>
  <tr align="center">
    <td>$\Large 01$</td>
    <td>$\Large K$</td>
  </tr>
  <tr align="center">
    <td>$\Large 1-$</td>
    <td>$\Large L$</td>
  </tr>
</table>

</div>

Il circuito ha due uscite z e w, che valgono 00 quando viene riconosciuta la sequenza JKJ, 01 quando viene riconosciuta la sequenza JKL, 10 quando viene riconosciuta la sequenza JLK, e 11 in tutti gli altri casi.

Sono ammesse sovrapposizioni.

> [!NOTE]
> Nella traccia del canale A-L viene richiesto di progettare solo il circuito sequenziale, di disegnare la tabella degli stati futuri ed il diagramma di transizione degli stati.
>
> Nella traccia del canale M-Z viene richiesto di progettare solo l'automa, di usare un FF di tipo SR per il bit più significativo e FF di tipo D per gli altri bit, di mostrare la tavola degli stati futuri, la tabella e il diagramma dell’automa e disegnare il circuito.

**Comment by M1SC0** [comment]
![image](https://github.com/user-attachments/assets/0317ddd9-bfdc-4eac-ba5b-9b07eb32fc62)
![image](https://github.com/user-attachments/assets/8e0de310-4d37-4e8a-bc02-30e5419db854)
